SHFB/Source/MRefBuilder/CCI/Metadata.cs

```diff
@@ -7,6 +7,8 @@
 // 11/23/2013 - EFW - Cleared out the conditional statements
 
 using System.Globalization;
+using System.IO;
+using System.IO.MemoryMappedFiles;

```

Comment:
MemoryMappedFiles using isn't needed, the one for IO is, but that path should be conditioned to only run on non-Windows as it loads the whole assembly in to a byte array